src/orchestratord/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+[package]
+name = "mz-orchestratord"
+description = "Kubernetes operator for Materialize regions"
+version = "0.120.0-dev.0"
+edition.workspace = true
+rust-version.workspace = true
+publish = false
+
+[lints]
+workspace = true
+
+[dependencies]
+anyhow = "1.0.66"
+async-trait = "0.1.68"
+axum = "0.7.5"
+chrono = { version = "0.4.35", default-features = false }
+clap = { version = "3.2.24", features = ["derive"] }
+futures = "0.3.25"
+http = "1.1.0"
+k8s-controller = "0.3.3"
+k8s-openapi = { version = "0.22.0", features = ["v1_29"] }
+kube = { version = "0.92.1", default-features = false, features = ["client", "runtime", "ws"] }
+maplit = "1.0.2"
+mz-alloc = { path = "../alloc" }
```

Comment:
It's a bit convoluted but you also need to depend on
```
mz-alloc-default = { path = "../alloc-default", optional = true }
```

And then the features section should be:
```
[features]
default = ["mz-alloc-default"]
jemalloc = ["mz-alloc/jemalloc"]
```

This will enable `jemalloc` by default, but make it opt-in on macOS. Take a look at [balancerd's Cargo.toml](https://github.com/MaterializeInc/materialize/blob/main/src/balancerd/Cargo.toml) for an example